processing/data_collection/gazette/pipelines.py

```diff
@@ -34,6 +34,9 @@ def process_item(self, item, spider):
         if not extract_text_from_file:
             return item
 
+        if len(item["files"]) == 0:
```

Comment:
I guess `not item["files"]` reads better, unless you want to check specifically for an empty list

Reply:
Fixed.